src/index.html

```diff
@@ -403,6 +406,7 @@ <h5>Aenean Ru Bristique</h5>
                 <a href="#" class="btn-outline"><i class="fas fa-exchange-alt"></i></a>
               </div>
               <div class="price">
+                <span>$ 45.00</span>
```

Comment:
Lepszym rozwiązaniem byłoby użycie klasy mówiącej, że jest to stara cena i użyć jej przy definiowaniu styli zamiast znacznika HTML.